<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QWidget" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>400</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#regButton {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(230, 80, 100, 230), stop:1 rgba(180, 0, 110, 230));
color:rgba(255,255,255,210);
border:1px solid rgba(150,0,85,255);
border-radius:5px;
}
QPushButton#regButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(250, 100, 120, 230), stop:1 rgba(200, 20, 130, 230));
}
QPushButton#regButton:pressed {
padding-left:5px;
padding-top:5px;
}
QPushButton#tutorialButton:hover {
color:rgba(220,150,160,255);
}
QPushButton#exitButton{
background-color:rgba(150,0, 100, 255);
}</string>
   </property>
   <widget class="QLabel" name="loginpanelLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>350</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/background.jpg);
border-radius:20px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="registerLabel">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>80</y>
      <width>185</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tokenLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(150,0,85,255);
color:rgba(255,255,255,240);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Token</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="symbolLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(150,0,85,255);
color:rgba(255,255,255,240);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Symbol</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pinLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(150,0,85,255);
color:rgba(255,255,255,240);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>PIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="githubButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/github-mark-white.png);
border-radius:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tutorialButton">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>415</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;</string>
    </property>
    <property name="text">
     <string>Where to find this data?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../f2a5ab68/res.qrc"/>
 </resources>
 <connections/>
</ui>
